env.dev/minio.env

```diff
@@ -1,6 +1,6 @@
 MINIO_ROOT_USER: minio99
 MINIO_ROOT_PASSWORD: minio123
-MINIO_IDENTITY_OPENID_CONFIG_URL: http://fusionauth:9011/.well-known/openid-configuration/265117b4-5065-42ca-895a-53625b89d380
+MINIO_IDENTITY_OPENID_CONFIG_URL: http://fusionauth:9011/.well-known/openid-configuration/f7d82f07-d34d-4201-98bb-aa8c3ff174b4
```

Comment:
Why this was changed ?

Reply:
The application ID was pre-defined in the .env. When I was testing the setup on my local, I created another application with this ID which made it's way to the repository. I noticed it while making the changes to readme and changed that to what it should Ideally be.